src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,116 @@
+import classNames from 'classnames';
+import React, { FC } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { Filter } from '../../types/Filter';
+import { getSearchWith } from '../../utils/searchHelper';
+import { SearchLink } from '../SearchLink';
+
+type Props = {
+  query: string,
+  sex: string | null,
+  centuries: string[],
+};
+
+export const PeopleFilters: FC<Props> = ({
+  query,
+  sex,
+  centuries,
+}) => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const centuryList = ['16', '17', '18', '19', '20'];
```

Comment:
It's better to save this constant outside the component to not create it on each render